layout/theme.liquid

```diff
@@ -31,6 +31,10 @@
     <script src="{{ 'constants.js' | asset_url }}" defer="defer"></script>
     <script src="{{ 'pubsub.js' | asset_url }}" defer="defer"></script>
     <script src="{{ 'global.js' | asset_url }}" defer="defer"></script>
+    <script src="{{ 'details-disclosure.js' | asset_url }}" defer="defer"></script>
+    <script src="{{ 'details-modal.js' | asset_url }}" defer="defer"></script>
+    <script src="{{ 'search-form.js' | asset_url }}" defer="defer"></script>
+
```

Comment:
I wonder here if we should also add: 
```liquid
{%- if settings.cart_type == "drawer" -%}
  <script src="{{ 'cart-drawer.js' | asset_url }}" defer="defer"></script>
{%- endif -%}
```
Which is in the header right now. So that if you click on `Add to cart` then the drawer is shown anyway?
If we're already including the styles assets, we might as well do the script too.

Reply:
Goooood point.  I just updated it - took a bit of a change to the code in drawer (since it has an awareness of the header), but wasn't too tricky.